dnf5/main.cpp

```diff
@@ -1444,6 +1444,11 @@ int main(int argc, char * argv[]) try {
 
     log_router.info("DNF5 finished");
 
+    // Print Complete! message only when transaction is created to prevent poluting output from repoquery or other command
+    if (auto * transaction = context.get_transaction(); transaction && !transaction->empty()) {
+        std::cout << "Complete!\n";
```

Comment:
Do we want to print the message also in quiet (`-q`) mode? If not, use `context.print_info("Complete");` instead of printing directly to `std::cout`.

In dnf5 quiet mode only partially works. For example, transaction table and transaction progress is visible even in quiet mode. We probably want to fix that as well. But that's out of the scope of this PR.

Reply:
Good point, I am changing it to react to `-q`.

Reply:
Anyway scriptlets are still visible in `-q` mode and even worse,  the new line is not added after the last entry.